db/migrate/20190925065737_add_email_field_to_solidus_gdpr_request.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class AddEmailFieldToSolidusGdprRequest < ActiveRecord::Migration[5.1]
+  def change
+    add_column :spree_gdpr_requests, :email, :string
+
+    reversible do |dir|
+      dir.up do
+        Spree::GdprRequest.all.each do |gdpr_request|
```

Comment:
Remember to always use `.find_each` instead of `.each` for AR collections. Also, I suggest wrapping this block in `ActiveRecord::Base.uncached { ... }`, so that AR doesn't cache the records, preventing OOM errors.